### ISSUE TITLE
Predict Error

### ISSUE THREAD

**Issue by bshim181** [zhpn1024/ribotish#40]
I inputted candidate ORF text file to narrow down the list of ORFs of interest with the TI Seq data. 
I am receiving this error after running predict. 

<img width="1235" alt="Screenshot 2024-11-18 at 9 27 04 AM" src="https://github.com/user-attachments/assets/e9200d4f-71da-49f7-a6af-d0b5146a2080">

Candidate ORFs input file looks something like this. 
<img width="1373" alt="Screenshot 2024-11-18 at 9 29 20 AM" src="https://github.com/user-attachments/assets/d8a2a77a-73fb-4b6a-937c-fe89c88d7acc">

I was wondering what might be the underlying cause for the error. Thank you!

**Comment by zhpn1024** [comment]
The input position need to be the relative position within the transcript length.

**Comment by bshim181** [comment]
Do you have a script that you would recommend to achieve this? 
I was wondering what about 5'UTR or 3'UTR ORFs? How would you expect those ORFs to be converted to transcript positions?

**Comment by zhpn1024** [comment]
The full length of transcript includes 5'UTR and 3'UTR. The position 0 is the first base of 5'UTR.
I can add an option to support genome start and end positions like your input file.

**Comment by bshim181** [comment]
That would be wonderful. do you have an idea of when that could be implemented by?

**Comment by bshim181** [comment]
<img width="242" alt="Screenshot 2024-11-18 at 11 03 37 PM" src="https://github.com/user-attachments/assets/400ad27d-90dd-4e17-a19a-7d95e4af9e9a">

i am giving it a try but facing this error.

**Comment by zhpn1024** [comment]
What's the genome position of ENSMUST...86843.1, are 484..1276 in this range?

**Comment by bshim181** [comment]
Sorry! I was running with the wrong candidate ORF file. It works now!

**Comment by zhpn1024** [comment]
Great!

**Comment by bshim181** [comment]
<img width="1045" alt="Screenshot 2024-11-20 at 1 39 22 AM" src="https://github.com/user-attachments/assets/933f9bff-6b14-4a29-8191-e0e4a805a324">

Hello, I do see this error appear again with the genome transcript. Could there be an issue if the ORFs start within the intron of the transcript?

**Comment by zhpn1024** [comment]
Yes, the intron positions are considered not in the transcript. They cannot be mapped to the transcript positions.
